tests/foreman/cli/test_repository.py

```diff
@@ -2168,16 +2171,16 @@ def test_module_stream_info_validation(self, repo):
             key: value for key, value in actual_result.items() if key in expected_result
         }
 
-    @pytest.mark.stubbed
     @pytest.mark.tier1
-    def test_negative_update_red_hat_repo():
+    @pytest.mark.skip_if_open('BZ:2002653')
+    def test_negative_update_red_hat_repo(self, module_manifest_org):
         """Updates to Red Hat products fail.
 
         :id: d3ac0ea2-faab-4df4-be66-733e1b7ae6b4
 
         :customerscenario: true
 
-        :BZ: 1756951
+        :BZ: 1756951,2002653
 
         :Steps:
             1. Import manifest and enable a Red Hat repository.
```

Comment:
2. Attempt to update the Red Hat repository.

Reply:
Noted